create-logux-creator/index.d.ts

```diff
@@ -93,13 +98,13 @@ export interface ReduxStateListener<S, A extends Action> {
 export class LoguxReduxStore<
   S = any,
   A extends Action = AnyAction,
-  H extends object = {},
+  C extends Client = Client<{}, Log<ClientMeta>>,
```

Comment:
Since `C` became completely optional, let’s move it to the last position, since developers will need a way to set `L`